typescript/packages/subsurface-viewer/src/components/Map.tsx

```diff
@@ -1591,32 +1608,101 @@ function buildDeckGlViews(views: ViewsType | undefined, size: Size): View[] {
     return result;
 }
 
+/**
+ * Returns true if the camera zoom is set.
+ * @param camera to be tested
+ * @returns true if the camera camera zoom is set.
+ */
+function cameraHasZoom(camera: ViewStateType | undefined): boolean {
+    return typeof camera?.zoom === "number" && !Number.isNaN(camera.zoom);
```

Comment:
`zoom` can be `[number, number]` in case of vertical scaling in orthographic view.

Reply:
Too bad there is no doc reflecting it in `MapProps`, nor story :/
Btw, it seems that `computeCameraZoom` does not handle the zoom correctly...

Since this PR is to fix a critical bug, can we complete it and add a follow-up to handle the `[number, number]` case ?